integration_tests/e2e/overview/goal/completeGoal.cy.ts

```diff
@@ -0,0 +1,152 @@
+import Page from '../../../pages/page'
+import OverviewPage from '../../../pages/overview/OverviewPage'
+import { putRequestedFor } from '../../../mockApis/wiremock/requestPatternBuilder'
+import { urlEqualTo } from '../../../mockApis/wiremock/matchers/url'
+import { matchingJsonPath } from '../../../mockApis/wiremock/matchers/content'
+import GoalsPage from '../../../pages/goal/GoalsPage'
+import CompleteOrArchiveGoalPage from '../../../pages/goal/CompleteOrArchiveGoalPage'
+import CompleteOrArchiveGoalValue from '../../../../server/enums/CompleteOrArchiveGoalValue'
+import CompleteGoalPage from '../../../pages/goal/CompleteGoalPage'
+
+context('Complete a goal', () => {
+  const prisonNumber = 'G6115VJ'
+  const goalReference = '10efc562-be8f-4675-9283-9ede0c19dade'
+
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignInAsUserWithEditAuthority')
+    cy.task('stubAuthUser')
+    cy.task('stubGetHeaderComponent')
+    cy.task('stubGetFooterComponent')
+    cy.task('stubPrisonerList')
+    cy.task('stubCiagInductionList')
+    cy.task('stubActionPlansList')
+    cy.task('getPrisonerById')
+    cy.task('stubGetInduction')
+    cy.task('getActionPlan')
+    cy.task('stubLearnerProfile')
+    cy.task('stubLearnerEducation')
+    cy.task('archiveGoal')
```

Comment:
I think you want this line here (instead of stubbing `archiveGoal`)
```
cy.task('completeGoal', { prisonNumber, goalReference })
```
And then you don't need it on lines 80 and 115